src/index.js

```diff
@@ -0,0 +1,115 @@
+import checkLocalStorage from './checkLocalStorage';
```

Comment:
## Modularise this module
- [ ] Spread this code in modules. Ensure `DOM-Manipulation` and `Logic` are in separate modules ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))